scripts/archive.sh

```diff
@@ -3,7 +3,7 @@ set -euo pipefail
 
 workspace=$1
 workspace_dir=$HIGH_PRIVACY_STORAGE_BASE/workspaces/$workspace
-archive=$HIGH_PRIVACY_STORAGE_BASE/archives/$workspace.tar.xz
+archive=/archive/archived_workspaces/$workspace.tar.gz
```

Comment:
Hmm, I think we should probably chance the mount point from /archive to /src/high_privacy/archives, so it's all under one dir, but that can be a different thing

Reply:
https://github.com/opensafely-core/sysadmin/issues/116